infra/base-images/base-clang/Dockerfile

```diff
@@ -23,7 +23,7 @@ ARG arch=x86_64
 ARG introspector
 ENV INTROSPECTOR_PATCHES=$introspector
 ENV FUZZINTRO_OUTDIR=$SRC
-ENV FUZZ_INTROSPECTOR=$introspector
```

Comment:
How will FUZZ_INTROSPECTOR be set anymore for checkout_and_build_clang to be used?